src/main/java/io/github/fixitlater/quizapi/QuestionController.java

```diff
@@ -92,10 +92,10 @@ public QuestionController(QuestionService questionService) {
         try {
             id = Long.parseLong(questionId);
         } catch (NumberFormatException e){
-            return null;
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
```

Comment:
Możecie zastanowić się nad alternatywnym podejściem do obsługi błędów jakie oferuje Spring. Tutaj jest kilka przykładów:
https://www.baeldung.com/exception-handling-for-rest-with-spring

O tyle może być to przydatne, że logikę obsługi strony z błędem piszecie raz, a w różnych miejscach aplikacji jedyne co robicie to rzucacie dany wyjątek. Ogranicza to ilość pisanego kodu i poprawia czytelność.